src/components/variable.jsx

```diff
@@ -28,18 +28,18 @@ const Variable = React.createClass({
         <div>
           <header className="page-header">
             <h1><code>{variable.id}</code></h1>
-            <p>{variable.description}</p>
+            <p className="descriptionForVariables">{variable.description}</p>
           </header>
           {this.renderVariableMetadata(variable)}
           <div>
             <ExternalLink href={variable.source}>
-              Code source de cette variable
+              <FormattedMessage id='editThisVariable'/>
```

Comment:
Be consistent on HTML attributes quotes : it seems that in the repository we use `"`, and not `'`.